community/cypher/cypher/src/test/scala/org/neo4j/cypher/ExecutionEngineTestSupport.scala

```diff
@@ -76,25 +65,32 @@ trait ExecutionEngineTestSupport extends CypherTestSupport with TestFriendlyExec
   }
 }
 
-trait TestFriendlyExecutionEngine {
+trait ExecutionEngineHelper {
+  self: GraphIcing =>
 
-  implicit class RighExecutionEngine(engine: ExecutionEngine) {
+  def graph: GraphDatabaseCypherService
 
-    def execute(query: String): ExtendedExecutionResult = {
-      engine.execute(query, Map.empty[String, Any], QueryEngineProvider.embeddedSession)
-    }
```

Comment:
It is not gone, it is here with a slightly different signature: https://github.com/neo4j/neo4j/pull/6551/files#diff-454fbfdc6f13cd0c8220fff1cd6f205cR75

The empty map has been added in all the places where we don't use the `ExecutionEngineTestSupport`